cmd/core/main.go

```diff
@@ -226,11 +228,23 @@ func main() {
 		setupLog.Error(err, "unable to create action client getter")
 		os.Exit(1)
 	}
+
+	aeClient, err := v1.NewForConfig(cfg)
```

Comment:
Nit
```suggestion
	aeClient, err := apiextensionsv1client.NewForConfig(cfg)
```

Reply:
updated in https://github.com/operator-framework/rukpak/pull/873/commits/3d4d5b5c90e98dceb74874d3796c0028167c4327